File: unit6/mario/mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")
    if height >= 1 and height <= 8:
        break

x = 0
for x in range(height):
    y = 0
    z = height - 1
    while z > x:
        print(" ", end="")
        z = z - 1
    while y <= x:
        print("#", end="")
        y = y + 1
    x = x + 1
    print("\n", end="")

File: unit3/examples/mario.c
#include <stdio.h>
#include <cs50.h>

void draw(int height);
int main(void)
{
    int n = get_int("n: ");
    draw(n);
}

void draw(int height)
{
    if (height == 0)
    {
        return;
    }
    draw(height - 1);

    for (int i = 0; i < height; i = i + 1)
    {
        printf("#");
    }
    printf("\n");
}